src/sentry/replays/usecases/ingest/dom_index.py

```diff
@@ -114,6 +114,30 @@ def create_replay_actions_payload(
     }
 
 
+def log_canvas_size(
+    org_id: int,
+    project_id: int,
+    replay_id: str,
+    events: list[dict[str, Any]],
+):
+    for event in events:
```

Comment:
Any concern with iterating through all events for every ingested segment?

Reply:
Nah, its a couple microseconds.  Parsing the JSON is the hard part and we're already doing that any way.